src/epcpm/parameterstointerface.py

```diff
@@ -1472,15 +1494,22 @@ def create_common_initializers(
         "true" if reject_from_inactive_interfaces else "false"
     )
 
+    scale_factor_variable_ref = "NULL"
+    if scale_factor_variable is not "NULL":
```

Comment:
Short strings tend to be interned and thus `is` works, but it is not guaranteed.

```suggestion
    if scale_factor_variable != "NULL":
```